exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillSimpleFuncHolder.java

```diff
@@ -40,10 +40,12 @@
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillSimpleFuncHolder.class);
 
   private final String drillFuncClass;
+  private final ClassLoader classLoader;
```

Comment:
Please insert a comment explaining the purpose of the class loader. As we know, the use is a bit of a trick when loading dynamic jars, so future readers will want some hints for how things work.